### PR TITLE
Fix moist air2 and apply the style guide to the next paragraph

### LINKED ISSUE #2963: Redeclaration from false class in example
In example listing _package MoistAir2_ (Sect. 7.3.1 The class extends Redeclaration Mechanism), the `model MoistAir_dynamicViscosity` extends from `PartialMedium.dynamicViscosity` which actually is a function. But this is prohibited according to Sect. 7.1.3 Restrictions on the Kind of Base Class.

### PR DESCRIPTION
**Pull request by HansOlsson**
Closes #2963 by changing it to a function.

### CODE REVIEW

File: chapters/introduction.tex [hunk 1]
```diff
@@ -22,6 +22,9 @@ \section{Scope of the Specification}\label{scope-of-the-specification}
 verified for those classes, before using them to build the simulation
 model.
 
+The restrictions for a simulation model apply when simulating the model.
+Thus a tool may translate a model lacking some key parts -- provided they are given when simulating the model.
+
```
Comment (henrikt-ma) [inline chapters/introduction.tex:27]: This belongs to a different issue, right?

File: chapters/inheritance.tex [hunk 2]
```diff
@@ -768,15 +768,22 @@ \subsection{The class extends Redeclaration Mechanism}\label{the-class-extends-r
...
+Besides the drawback that the namespace is polluted by elements that have different names but the same implementation (e.g.\ \lstinline!MoistAir2.BaseProperties! is identical to \lstinline!MoistAir2.MoistAir_BaseProperties!) the whole construction does not work if arrays are present that depend on constants in \lstinline!PartialMedium!, such as \lstinline!X[nX]!:
+The problem is that \lstinline!MoistAir_BaseProperties! extends from \lstinline!PartialMedium.BaseProperties! where the constant \lstinline!nX! does not yet have a value.
+This means that the dimension of array \lstinline!X! is undefined and model \lstinline!MoistAir_BaseProperties! is wrong.
+With this construction, all constant definitions have to be repeated whenever these constants shall be used, especially in \lstinline!MoistAir_BaseProperties! and \lstinline!MoistAir_dynamicViscosity!.
+For larger models this is not practical and therefore the only practically useful definition is the complicated construction in the previous example with \lstinline!redeclare model extends BaseProperties!.
```
Comment (henrikt-ma) [inline chapters/inheritance.tex:786]: I'm trying to spot the change here.  Is there anything in addition to the sentence-based line breaks?

**Review by henrikt-ma — CHANGES_REQUESTED** [review]
The last commit looks like a fix for the issue to be closed, but the other three commits are probably included in the PR just by mistake.

**Review by eshmoylova — COMMENTED** [review]
I agree with @henrikt-ma's comments. Maybe it's worth updating the description of the pull request to "Fix moist air2 and apply the style guide to the next paragraph" or something like that to make it explicit?

File: chapters/inheritance.tex (hunk 2 above)
Comment (HansOlsson) [reply chapters/inheritance.tex:786]: No, just breaking the lines.

File: chapters/introduction.tex (hunk 1 above)
Comment (HansOlsson) [reply chapters/introduction.tex]: Yes, removed.

**Comment by HansOlsson** [comment]
> I agree with @henrikt-ma's comments. Maybe it's worth updating the description of the pull request to "Fix moist air2 and apply the style guide to the next paragraph" or something like that to make it explicit?

Done.

**Review by eshmoylova — APPROVED** [review]

**Review by henrikt-ma — APPROVED** [review]
Looks like all problems have been addressed.

**Pull request merged**